builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -1290,6 +1334,19 @@ func normalizeRegion(region string) string {
 	return region
 }
 
+func validateS3BucketAccelerationStatus(v interface{}, k string) (ws []string, errors []error) {
+	value := strings.ToLower(v.(string))
+	validTypes := map[string]struct{}{
+		"Enabled":   struct{}{},
+		"Suspended": struct{}{},
```

Comment:
`TestAccAWSS3Bucket_acceleration` fails for me, because we lowercase the value here but the `validTypes` are capitalized... odd